DOCS-2313: Improve notification URL for ecommerce integrations page (#1848)
MultiSafepay/docs: content/account/api-onboarding/add-websites.md

4. [Update website](#update-a-website): Update a website.

### About parameters

> For every parameter, we specify whether it's _required_ or _optional_. However, this label refers only to the technical requirements for valid API requests. To process credit card transactions with the website [API key](/set-up-your-account/site-id-api-key-secure-code), expected minimum and maximum order values need to be supplied.

Reviewer: ```suggestion
We specify whether each parameter is _required_ or _optional_ for the request to be valid. For example,  to process a credit card payment with a website [API key](/set-up-your-account/site-id-api-key-secure-code), you must provide your expected minimum and maximum order values.
```

> For every parameter, we specify whether it's _required_ or _optional_. However, this label refers only to the technical requirements for valid API requests. To process credit card transactions with the website [API key](/set-up-your-account/site-id-api-key-secure-code), expected minimum and maximum order values need to be supplied.

Author: Agreed. I've removed the paragraph.
